### PR TITLE
Tweak OpenAPI schema

### PR DESCRIPTION
**Pull request by mvdbeek**
- Drop duplicate operation tags
- Fix inconsistent path parameter names

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/api/history_contents.py [hunk 1]
```diff
@@ -511,22 +511,22 @@ def download_dataset_collection(
     @router.post(
         "/api/histories/{history_id}/contents/dataset_collections/{id}/prepare_download",
         summary="Prepare an short term storage object that the collection will be downloaded to.",
+        include_in_schema=False,
+    )
+    @router.post(
```
Comment (jmchilton) [inline lib/galaxy/webapps/galaxy/api/history_contents.py:516]: I have an alternate fix for this... in a branch that will be very conflicted 😆. hackathons

**Comment by jmchilton** [comment]
Fixed a relevant API test failure based on the parameter name change. Thanks for disambiguation fixes - they are very nice!

**Comment by jmchilton** [comment]
Ekkk - sorry about the conflicting typo fix.

**Comment by mvdbeek** [comment]
No worrries, doing it all at once is great!

**Pull request merged**